{"name": "test_invalid_login", "status": "failed", "statusDetails": {"message": "AssertionError: Expected error message not shown\nassert '' != ''", "trace": "driver = <appium.webdriver.webdriver.WebDriver (session=\"c1ee63221a0a08110db6a4f88b79d409437df47c\")>\n\n    def test_invalid_login(driver):\n        lp = LoginPage(driver)\n        # known invalid user in sample app\n        lp.login(\"locked_out_user\", \"secret_sauce\")\n        err = lp.get_error()\n>       assert err != \"\", \"Expected error message not shown\"\nE       AssertionError: Expected error message not shown\nE       assert '' != ''\n\ntests\\test_login.py:16: AssertionError"}, "attachments": [{"name": "screenshot-android", "source": "2c617dbf-94ff-4302-94a2-aa243b30ed91-attachment.png", "type": "image/png"}], "start": 1757858676507, "stop": 1757858684607, "uuid": "b52a3b5e-48c8-482e-a64d-69b465bc258f", "historyId": "6485cd1aa206d185bd707f422a9a9238", "testCaseId": "6485cd1aa206d185bd707f422a9a9238", "fullName": "tests.test_login#test_invalid_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DB-L-174"}, {"name": "thread", "value": "15608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}